cli/tests/testdata/bench/multifile_summary.out

```diff
@@ -37,6 +39,7 @@ bench7 [WILDCARD] [WILDCARD]/iter[WILDCARD]([WILDCARD] … [WILDCARD]) [WILDCARD
 bench8 [WILDCARD] [WILDCARD]/iter[WILDCARD]([WILDCARD] … [WILDCARD]) [WILDCARD]
 bench9 [WILDCARD] [WILDCARD]/iter[WILDCARD]([WILDCARD] … [WILDCARD]) [WILDCARD]
 
+
 [WILDCARD]/bench/group_baseline.ts
 benchmark           time (avg)        iter/s             (min … max)       p75       p99      p995
 -------------------------------------------------------------------- -----------------------------
```

Comment:
Could you modify this test to use two different groups (ie. in addition to `url` group, add `noop` group).

Reply:
Done.